app/javascript/app/components/emission-pathways-scenario-table/emission-pathways-scenario-table-selectors.js

```diff
@@ -12,13 +17,60 @@ const getIdData = createSelector([getData, getId], (data, id) => {
   return data.find(d => String(d.id) === id) || null;
 });
 
-const getIndicatorsData = createSelector(getIdData, data => {
+const getIndicatorIds = createSelector(getIdData, data => {
   if (!data) return null;
-  return data.indicators || null;
+  return data.indicators.map(i => i.id) || null;
 });
 
-export const filteredDataBySearch = createSelector(
-  [getIndicatorsData, getQuery],
+const scenarioIndicatorsData = createSelector(
+  [state => state.espIndicatorsData, getIndicatorIds],
+  (indicatorsData, ids) => {
+    if (!ids || isEmpty(indicatorsData) || isEmpty(ids)) return null;
+    return indicatorsData.filter(i => i.id) || null;
+  }
+);
+
+export const getCategories = createSelector(scenarioIndicatorsData, data => {
+  if (!data) return null;
+  return uniq(data.map(i => i.category.name || i.category)).map(c => ({
+    value: c,
+    label: c
+  }));
+});
+
+export const getSelectedCategoryOption = createSelector(
```

Comment:
A simple name (to match all the other selectors in the site) getCategorySelected? Sorry I know now I am being picky :)